### PR TITLE
docs: minor edits to Teleport Connect guide

### PR DESCRIPTION
**Pull request by stevenGravy**
- defines Teleport Connect as a desktop GUI app
- defines `tsh` tool what some folks may not know about
- provides link to `tsh` tool guide
- uses `tsh` formatting consistent with `tsh` guide

### CODE REVIEW

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 1]
```diff
@@ -3,8 +3,8 @@ title: Using Teleport Connect
 description: Using Teleport Connect
 ---
 
-Teleport Connect provides easy and secure access to SSH servers, databases, and Kubernetes clusters,
-with support for other resources coming in the future.
+Teleport Connect provides easy and secure access to SSH servers, databases, and Kubernetes clusters
+as an intuitive desktop GUI application, with support for other resources coming in the future.
```
Comment (zmb3) [inline docs/pages/connect-your-client/teleport-connect.mdx:7]: Intuitive is subjective. Let's rephrase this. Maybe something like

_Teleport Connect is a desktop app that provides secure access ..._

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 2]
```diff
@@ -130,24 +130,25 @@ progress when the app was closed. It will only restore the working directory for
...
 a custom home location through [the `TELEPORT_HOME` environment
 variable](../reference/cli.mdx#tsh-environment-variables). For example, logging in to a new cluster
-through tsh will not make that cluster show up in Teleport Connect.
+through `tsh` will not make that cluster show up in Teleport Connect.
 
 <Tabs>
 <TabItem label="macOS">
-To add tsh to `PATH`, execute `tsh install` from the command bar in Teleport Connect. This will
-symlink tsh to `/usr/local/bin`. You can remove the symlink with `tsh uninstall`.
+To add `tsh` to your user `PATH`, execute `tsh install` from the command bar in Teleport Connect. This will
```
Comment (zmb3) [inline docs/pages/connect-your-client/teleport-connect.mdx:145]: What's a _user PATH_? Is this extra verbiage necessary?

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 3]
```diff
@@ -130,24 +130,25 @@ progress when the app was closed. It will only restore the working directory for
...
+symlink `tsh` to `/usr/local/bin`. You can remove the symlink with `tsh uninstall`.
 
-If you used the tsh macOS .pkg installer before, this will overwrite the symlink made by that installer.
+If you installed the `tsh` macOS .pkg installer already, this will overwrite the symlink made by that installer.
```
Comment (zmb3) [inline docs/pages/connect-your-client/teleport-connect.mdx:148]: This sentence says install 3 times - can we remove some of the redundancy?

**Comment by alexfornuto** [comment]
@stevenGravy Just checking in here. Looks like there are outstanding comments (and merge conflicts) that should be resolved before the Docs team takes a look.

**Comment by ptgott** [comment]
@stevenGravy Do you still plan to move forward with this one?

**Comment by ptgott** [comment]
Hi @stevenGravy, are we still moving ahead with this PR?

**Comment by ptgott** [comment]
@stevenGravy Following up on this one

**Comment by zmb3** [comment]
Closing due to inactivity.

**Pull request closed without merging**